심화요구사항.md

```
@@ -0,0 +1,29 @@
+
+- [x]  Repository 구현체 중에 어떤 구현체를 Bean으로 등록할지 Java 코드의 변경 없이 application.yaml 설정 값을 통해 제어해보세요.
+
+```yaml
+# application.yaml
+discodeit:
+  repository:
+    type: jcf   # jcf | file
+```
+
+- [x] discodeit.repository.type 설정값에 따라 Repository 구현체가 정해집니다.
+    - [x] 값이 jcf 이거나 없으면 JCF*Repository 구현체가 Bean으로 등록되어야 합니다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)